test/system/helpers.network.bash

```diff
@@ -251,6 +251,7 @@ function port_is_bound() {
     *"."*)
         grep -e "^[^:]*: $(ipv4_to_procfs "${address}"):${port}"    \
              -e "^[^:]*: $(ipv4_to_procfs "0.0.0.0"):${port}"       \
+             -e "^[^:]*: $(ipv4_to_procfs "127.0.0.1"):${port}"     \
```

Comment:
Personally, I think this is wrong: I think this code (or perhaps `ipv4_to_procfs`?) should special-case `0.0.0.0`, converting it to `[0-9A-F]+` instead. But I'm overwhelmed with backlog right now, and don't have time/interest in finding the perfect solution. @sbrivio-rh your thoughts/suggestions welcome.